src/defs.js

```diff
@@ -288,14 +288,10 @@ if (WEB) {
     const LUA_CDIR = "./lua/" + LUA_VDIR + "/";
     module.exports.LUA_CDIR = LUA_CDIR;
 
-    const LUA_PATH_DEFAULT =
-        LUA_LDIR + "?.lua;" + LUA_LDIR + "?/init.lua;" +
-        LUA_CDIR + "?.lua;" + LUA_CDIR + "?/init.lua;" +
-        "./?.lua;./?/init.lua";
+    const LUA_PATH_DEFAULT = "?.lua;?/init.lua";
     module.exports.LUA_PATH_DEFAULT = LUA_PATH_DEFAULT;
 
-    const LUA_CPATH_DEFAULT =
-        LUA_CDIR + "?.js;" + LUA_CDIR + "loadall.js;./?.js";
+    const LUA_CPATH_DEFAULT = "?.js;loadall.js";
```

Comment:
howso? You could have set `LUA_CDIR` to the empty string to get the same effect as changing this line.
But even then: why have this change at all?